ui/app/serializers/oidc/provider.js

```diff
@@ -2,4 +2,15 @@ import ApplicationSerializer from '../application';
 
 export default class OidcProviderSerializer extends ApplicationSerializer {
   primaryKey = 'name';
+
+  normalizeItems(payload) {
+    if (payload.data) {
+      if (payload.data?.keys && Array.isArray(payload.data.keys)) {
```

Comment:
I'm assuming if this condition isn't met there is no data returned or there is an error returned?

Reply:
If that condition isn't met the original payload passed without the data would get returned from line 14 and line 13 deletes the data from the payload which contains the extra issuer and other metadata information.